README.md

```diff
@@ -87,9 +87,9 @@ console.log(Anser.ansiToJson(txt));
 ```
 
 
-When using **Typescript** you can do the following:
+When using **TypeScript** you can do the following:
```

Comment:
Can you please edit the [`package.json`](https://github.com/IonicaBizau/anser/blob/277efe80d250f5a52b426dd224f4ef0d334ae8f8/package.json#L49) content too?

Reply:
Ah, sorry, didn't realize that it was in package.json as well. Should be fixed now